src/core/DY.Core.Bytes.fst

```diff
@@ -3065,3 +3125,133 @@ let bytes_invariant_kem_decap #ci tr sk encap =
     else ()
   | _ -> ()
 #pop-options
+
+(*** MAC ***)
+
+/// Constructor.
+
+[@@"opaque_to_smt"]
+val mac_auth: bytes -> bytes -> bytes
+let mac_auth key msg =
```

Comment:
Minor: `mac_auth` seems like it might be a bit unintuitive of a name --- `mac_sign` matches `mac_verify`, but I also hear MACs referred to as tags often enough that `mac_tag` would also sound reasonable to me.

Reply:
I agree `mac_auth` is not a great name, but I don't like `mac_sign` either (confusing with respect to signatures) or `mac_tag` (feels cryptic).

What about `mac_compute`?

Reply:
I think `mac_compute` is fine.

Reply:
`mac_compute` is less confusing than `mac_auth`. Maybe `mac_compute_tag` is a bit more explicit, even, but `mac_compute` is probably enough improvement.

Reply:
I changed to `mac_compute` then